### ISSUE TITLE
BorderSpacing should not be a number

### ISSUE THREAD

**Issue by droda59** [handsontable/handsontable#5291]
### Description
In the `detectCaptionProblem` function, a table element is created with default values for border spacing, width and padding. 
Following CSSStyle, these values should be strings, not numbers. 
There is a problem with Jest when a HotTable is generated since it goes through this element creation, and Jest does a `v.toLowerCase()` on the border-spacing (which is a number at that point). 

The `detectCaptionProblem` function should assign values to the style like this: 
```
  TABLE.style.borderSpacing = '0';
  TABLE.style.borderWidth = '0';
  TABLE.style.padding = '0';
```

### Your environment
* Handsontable version: 5.0.0
* Jest version: 23.4.1

**Comment by AMBudnik** [comment]
Thank you for sharing the investigation results @droda59 

I have asked my colleague to check the issue on his implementation.

**Comment by wojciechczerniak** [comment]
Some other tool does not support value other than string. I can't find any source stating that the value `0` is invalid. Do you have any link @droda59? 

@jansiegel Might be related to handsontable/vue-handsontable-official/issues/65 jsakas/CSSStyleDeclaration/issues/63 and jsakas/CSSStyleDeclaration/pull/52 and already resolved.

**Comment by jansiegel** [comment]
According to https://www.w3.org/TR/DOM-Level-2-Style/css.html#CSS-CSSStyleDeclaration, we can assume, that the `CSSStyleDeclaration` interface expects the css property value to be a string, so I guess this is the way to go.

@wojciechczerniak It is. There are numerous places in our code, where we declare `0` as an integer and pass it to the `style` property of DOM elements, so I think we should try to correct all of them. The problem with it is that the integer declaration _technically_ works fine, so this bug might come back if we're not  vigilant enough.

**Comment by AMBudnik** [comment]
Hey @droda59 

I'm happy to announce that this fix has been released in v 6.2.1
Thank you for sharing.
